pydantic/main.py

```diff
@@ -700,7 +702,7 @@ def _get_value(
         value_type = type(v)
         if value_type in NOOP_TYPES:
             return v
-        if isinstance(v, BaseModel):
+        if value_type == BaseModel or isinstance(v, BaseModel):
```

Comment:
BaseModel is an abstract class?  I don't think this would actually help.

Reply:
Anything that's extending the BaseModel type would get short-circuited here right? Would this hurt?

Reply:
`value_type == BaseModel` would never be true because I think users of it always subclass